src/app/contacts/page.js

```diff
@@ -1,7 +1,71 @@
+import Image from 'next/image';
+
 const HomePage = () => {
   return (
-    <div className='bg-amber-200'>
-      <h1>Contacts page</h1>
+    <div className='xl:px-18 px-5 lg:px-12 2xl:px-40'>
+      <h1 className='decorationLine my-16'>Контакти</h1>
+
+      <div className='mb-10 grid grid-cols-1 justify-center gap-5 md:grid-cols-3 lg:flex'>
+        <div className='block-contacts'>
```

Comment:
I think it would be better to create an object with contacts data: 
```
const contactsData = [
{
  iconSrc = '/icons/market-map.svg',
  title: 'Location',
  text: 'Україна с. Колочава Закарпатська обл. Межигірський р-н.',
}, 
...
];
```
and then loop through the array of contacts like this for example:
```
{contactsData.map(contact => (
   <div className='block-contacts'>
      <Image src={contact.iconSrc} alt='icon' width={40} height={40} />
      <h3>{contact.title}</h3>
      <p className='text-center text-Green-400'>{contact.text}</p>
   </div>
)}
```